allennlp/commands/elmo.py

```diff
@@ -257,21 +247,13 @@ def embed_file(self,
             A path to the output hdf5 file.
         batch_size : ``int``, optional, (default = 64)
             The number of sentences to process in ELMo at one time.
-        use_sentence_key : ``bool``, optional, (default = False)
-            If true, use the first token in each line as the unique key for the layers output to the HDF5 file.
-            This key will be stripped from the rest of the line and the remaining tokens will be used as the
-            sentence to compute embeddings from.
         """
 
         # Tokenizes the sentences.
-        sentences = [line.strip().split() for line in input_file]
-        if use_sentence_key:
-            # Uses the first token in each sentence as the key.
-            keys, sentences = zip(*[(tokens[0], tokens[1:]) for tokens in sentences])
-            embedded_sentences = zip(keys, self.embed_sentences(sentences, batch_size))
-        else:
-            # Uses the index as the key.
-            embedded_sentences = enumerate(self.embed_sentences(sentences, batch_size))
+        sentences = [line.strip() for line in input_file]
+        split_sentences = map(lambda sentence: sentence.split(), sentences)
```

Comment:
`split_sentences = [sentence.split() for sentence in sentences]`

Reply:
Thanks.  Writing that map did feel awkward...